tests/sympc/protocol/falcon/falcon_test.py

```diff
@@ -333,3 +333,22 @@ def test_private_compare_exception(get_clients) -> None:
     # Exception for not passing a public value(torch.Tensor).
     with pytest.raises(ValueError):
         Falcon.private_compare([x], x)
+
+
+def test_wrap(get_clients) -> None:
```

Comment:
Let's add a test for malicious security type?

Reply:
Added